### DISCUSSION TITLE
Extra domains option

### DISCUSSION THREAD

**Discussion by ndTEC** [postalserver/postal#1877] [Help with using Postal]
Hi,

What is the purpose of the Domains option from Home page?

![p1](https://user-images.githubusercontent.com/79922948/155117067-7e2494d1-63a8-49ad-b071-563e28d59b6e.png)

**Comment by willpower232** [comment]
If you set up the domain at the top organization level, it will apply to all servers under it

this is handy if you want to have separate servers for different departments but access to the same domain or if you wanted a standard testing domain to apply to all of servers in that organization
